CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ CHANGELOG
 - Automation API - add recovery APIs (cancel/export/import)
   [#5369](https://github.com/pulumi/pulumi/pull/5369)
 
+- Add `pulumi console` command which opens the currently selected stack in the Pulumi console.
```

Comment:
This will need to be moved up to HEAD since 2.10.1 has already been released.

Reply:
Ahh yes, thank you.